.github/workflows/main.yml

```diff
@@ -34,15 +34,26 @@ jobs:
     - name: Test
       if: matrix.os != 'ubuntu-latest'
```

Comment:
On the neo-modules repo, tests were running just on " ubuntu-latest".
Since coverage is just on ubuntu-latests as well, I think that if there is any trouble we can push the fix in another PR.